SDKAndroid/src/main/java/app/cybrid/sdkandroid/util/DataAccessStrategy.kt

```diff
@@ -31,8 +31,6 @@ suspend fun <T> getResult(call: suspend() -> Response<T>): Resource<T> {
             Logger.log(LoggerEvents.AUTH_EXPIRED, "${response.code()} - ${response.message()}")
             return Resource.error(response.message(), code = response.code())
         } else {
-            Logger.log(LoggerEvents.NETWORK_ERROR,
-                "${response.code()} ${response.message()} :: ${response.raw()}")
```

Comment:
Why did we remove this?

Reply:
Was log duplicate errors, because in each transaction (inside ViewModels) we log the errors in error response network.